test/fixtures/users.yml

```diff
@@ -0,0 +1,12 @@
+user1:
+  username: userone
+user2:
+  username: usertwo
+user3:
+  username: userthree
+user4:
+  username: userfour
+user5:
+  username: userfive
+testuser:
+  username: testuser
```

Comment:
Very creative usernames.  😉 

(Nothing wrong with functional names; just thought it was funny.)